CHANGELOG.md

```diff
@@ -7,7 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-
+## [1.0.1]()
+
+values files changes:
```

Comment:
Changelog shouldn't include the complete description of each PR - we have the PR description and JIRA (assuming it will be open at some point) for people who would like to dive deeper into the change details.

Instead, Changelog entries should have one or two sentences max per each PR with a descriptive title that gives a quick hint of the change introduced in each PR.

Although we aren't following the Kubernetes' changelog format, but here's an example for the level of detail we want to keep in the Changelog: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#feature-1

Please also include a link to the PR at the end of each entry. For example:

```
### Added
- Bitnami Helm charts for Kafka, Cassandra and PostgreSQL ([#49](https://github.com/Telecominfraproject/wlan-cloud-helm/pull/49))
```

Reply:
While I'm at it, we also need to add a changelog entry for: https://github.com/Telecominfraproject/wlan-cloud-helm/pull/53

Reply:
do we want a single one for both of these or not?

Reply:
sorry, I meant a single release for both of these commits?